nixos/modules/services/logging/logstash.nix

```diff
@@ -88,7 +88,7 @@ in
         example = ''
           # Read from journal
           pipe {
-            command => "''${pkgs.systemd}/bin/journalctl -f -o json"
+            command => "${pkgs.systemd}/bin/journalctl -f -o json"
```

Comment:
I think `literalExample` would do what you want here. It's not clear to me that the original version is deficient, though.